polyglot/upload.py

```diff
@@ -122,7 +124,7 @@ def filter_utxos_for_bcat(self):
         """filters out all utxos with 0 conf or too low amount for use in a BCAT part transaction"""
         filtered_utxos = []
         for utxo in self.get_unspents():
-            if utxo.confirmations != 0 and utxo.amount >= 100000:
+            if utxo.confirmations >= self.utxo_min_confirmations and utxo.amount >= self.fee * 100000:
```

Comment:
This is the same story as below - The idea at the time was to keep it simple and always fix the utxo.amount to be a save amount depending on the amount of data in the data carrier tx. (Which was 100kb at the time...)...

So make it easy, I enforced that each bcat part should not exceed 90,000 bytes so that there is always 10,000 satoshis left over to cover whatever the fee is...

Would be nice to calculate it all down to the last satoshi somehow but each transaction will always be different so there's no way to know the ideal utxo.amount ahead of time anyway... so just went with (hopefully) somewhat sensible defaults to cover all cases.